feat : add-remove-endpoint

MaXM91 opened the pull request:
feat : 
- ввести возможность удаления фильмов и юзеров по ид.
    Добавить методы:
    - deleteUserById в UserController.java,
    - deleteById в UserService.java,
    - deleteFilm в FilmController.java,
    - deleteById в FilmService.java.
- провести тестирование методов.
    Добавить методы:
    - shouldDeleteFilmById() в FilmRepositoryTests.java,
    - shouldDeleteUserById() в UserRepositoryTests.java.

Commit:
feat :
- ввести возможность удаления фильмов и юзеров по ид.
    Добавить методы:
    - deleteUserById в UserController.java,
    - deleteById в UserService.java,
    - deleteFilm в FilmController.java,
    - deleteById в FilmService.java.
- провести тестирование методов.
    Добавить методы:
    - shouldDeleteFilmById() в FilmRepositoryTests.java,
    - shouldDeleteUserById() в UserRepositoryTests.java.

Commit:
feat :
- ввести возможность удаления фильмов и юзеров по ид.
    Добавить методы:
    - deleteUserById в UserController.java,
    - deleteById в UserService.java,
    - deleteFilm в FilmController.java,
    - deleteById в FilmService.java.
- провести тестирование методов.
    Добавить методы:
    - shouldDeleteFilmById() в FilmRepositoryTests.java,
    - shouldDeleteUserById() в UserRepositoryTests.java.

Commit:
feat :
- ввести возможность удаления фильмов и юзеров по ид.
    Добавить методы:
    - deleteUserById в UserController.java,
    - deleteById в UserService.java,
    - deleteFilm в FilmController.java,
    - deleteById в FilmService.java.
- провести тестирование методов.
    Добавить методы:
    - shouldDeleteFilmById() в FilmRepositoryTests.java,
    - shouldDeleteUserById() в UserRepositoryTests.java.

Commit:
feat :
- ввести возможность удаления фильмов и юзеров по ид.
    Добавить методы:
    - deleteUserById в UserController.java,
    - deleteById в UserService.java,
    - deleteFilm в FilmController.java,
    - deleteById в FilmService.java.
- провести тестирование методов.
    Добавить методы:
    - shouldDeleteFilmById() в FilmRepositoryTests.java,
    - shouldDeleteUserById() в UserRepositoryTests.java.

Commit:
feat :
- ввести возможность удаления фильмов и юзеров по ид.
    Добавить методы:
    - deleteUserById в UserController.java,
    - deleteById в UserService.java,
    - deleteFilm в FilmController.java,
    - deleteById в FilmService.java.
- провести тестирование методов.
    Добавить методы:
    - shouldDeleteFilmById() в FilmRepositoryTests.java,
    - shouldDeleteUserById() в UserRepositoryTests.java.

Commit:
feat :
- ввести возможность удаления фильмов и юзеров по ид.
    Добавить методы:
    - deleteUserById в UserController.java,
    - deleteById в UserService.java,
    - deleteFilm в FilmController.java,
    - deleteById в FilmService.java.
- провести тестирование методов.
    Добавить методы:
    - shouldDeleteFilmById() в FilmRepositoryTests.java,
    - shouldDeleteUserById() в UserRepositoryTests.java.

pesotskypv reviewed (approved):
Критических замечаний нет. Принято.

Commit:
feat :
- ввести вывод общих фильмов по ид 2 пользователь.
    Добавить методы:
    - getSharedFilms в FilmController.java,
    - getFilmsShared в FilmService.java,
    - foundFilmsShared в интерфейс FilmRepository.java,
    - foundFilmsShared в FilmRepositoryImpl.java,
    - foundFilmsShared в InMemoryFilmRepository.java.
- провести тестирование методов.
    Добавить метод:
    - shouldGetFilmsSharedTest() в FilmRepositoryTests.java.
